### PR TITLE
Completed javascript-methods exercise.

### PR DESCRIPTION
**Pull request by twoosam**
image

### CODE REVIEW

File: javascript-methods/main.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const num1 = 4;
+const num2 = 7;
+const num3 = 10;
+
+const maximumValue = Math.max(num1, num2, num3);
+console.log('maximumValue is:', maximumValue);
+
+const heroes = ['Iron Man', 'Captain America', 'Batman', 'Spiderman'];
+let randomNumber = Math.random(heroes);
```
Comment (shawnkost) [inline javascript-methods/main.js:9]: `Math.random` does not take any arguments. You shouldn't be passing anything in when you call this method

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Tim!

**Comment by twoosam** [comment]
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**